DBproject_part2 (2).py

```diff
@@ -0,0 +1,132 @@
+#My_project  imports -> function -> code
+def menu():
+    print("1. Save a new entry")
+    print("2. Search by ID")
+    print("3. Print ages average")
+    print("4. Print all names")
+    print("5. Print all IDs")
+    print("6. Print all entries")
+    print("7. Print entry by index")
+    print("8. Exit")
+
+def saveNewEntry(entries):
+    id_input = input("ID: ")
+    if not id_input.isdigit():
+        print("ERROR: Your ID can only contain numbers " + str(id_input) + " is not a number")
+        input("Press Enter to continue...")
+        return None
+    if id_input in entries:
+        print("ERROR: The ID " + str(id_input) + " already exists.")
+        input("Press Enter to continue...")            #creat function ToContinue 
+        return None

```

Comment:
אם את תחזירי 0 ולא None, את תוכלי לוותר על 
if age is not None: 
בפונקציה מיין.
הבעיה היחידה שתהיה לך היא עם age_count
אבל לא צריך age_count, כי יש לך את מספר האיברים, זה הlen של entries